### PR TITLE
feat: add a voting app that spins up using redis

### PR DESCRIPTION
**Pull request by h4ck3rk3y**

### CODE REVIEW

File: redis-voting-app/main.star [hunk 1]
```diff
@@ -0,0 +1,23 @@
+main_redis_module = import_module("github.com/kurtosis-tech/redis-package/main.star")
+
+VOTING_APP_IMAGE = "mcr.microsoft.com/azuredocs/azure-vote-front:v1"
```
Comment (leoporoli) [inline redis-voting-app/main.star:3]: great that you ping the version

File: redis-voting-app/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+```
+
+If you have this repository cloned locally and are playing with the package, use
+
+```
+cd redis-voting-app
+kurtosis run .
+```
+
+Once the package is up and running use the localhost binding of port 80 for the `voting-app`; like
```
Comment (leoporoli) [inline redis-voting-app/README.md:16]: WDYT about replacing `...of port 80` with `...of HTTP port` o `...of the API port`?

File: redis-voting-app/main.star [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+def run(plan, args):
+
+    plan.print("Spinning up the Redis Package")
+    redis_run_output = main_redis_module.run(plan, args)
+
+    redis_service_name = redis_run_output["hostname"]
```
Comment (leoporoli) [inline redis-voting-app/main.star:10]: shouldn't be `redis_hostname` or `redis_address` the var name ?

Comment (h4ck3rk3y) [reply redis-voting-app/main.star:10]: You are right; I was using service name earlier from output but then switched to hostname to show a feature + for it to work in k8s! 👍

File: cassandra-network-partition-test/main.star [hunk 4]
```diff
@@ -40,7 +40,7 @@ def simulate_network_failure(plan, cassandra_run_output):
         command = ["/bin/sh", "-c", check_un_nodes],
     )
 
-    plan.wait(check_un_nodes_recipe, "output", "==", str(len(cassandra_run_output["node_names"])-1))
+    plan.wait(check_un_nodes_recipe, "output", "==", str(len(cassandra_run_output["node_names"])-1), timeout="5m")
```
Comment (leoporoli) [inline cassandra-network-partition-test/main.star:43]: nit: could you use `named arguments` in all arguments here? that will help users to understand the code

File: cassandra-network-partition-test/main.star [hunk 5]
```diff
@@ -73,4 +73,4 @@ def heal_and_verify(plan, cassandra_run_output):
         command = ["/bin/sh", "-c", node_tool_check],
     )
 
-    plan.wait(check_nodes_are_up, "output", "==", str(len(cassandra_run_output["node_names"])))
+    plan.wait(check_nodes_are_up, "output", "==", str(len(cassandra_run_output["node_names"])), timeout="5m")
```
Comment (leoporoli) [inline cassandra-network-partition-test/main.star:76]: same here

**Pull request merged**